lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -387,19 +490,26 @@ class MetricsEndpointConsumer(Object):
 
     on = MonitoringEvents()
 
-    def __init__(self, charm, name):
+    def __init__(self, charm: CharmBase, relation_name: str = DEFAULT_RELATION_NAME):
         """A Prometheus based Monitoring service provider.
 
         Args:
             charm: a `CharmBase` instance that manages this
                 instance of the Prometheus service.
-            name: string name of the relation over which scrape target
-                information is gathered by the Prometheus charm.
+            relation_name: an optional string name of the relation between `charm`
+                and the Prometheus charmed service. The default is "metrics-endpoint".
+                It is strongly advised not to change the default, so that people
+                deploying your charm will have a consistent experience with all
+                other charms that consume metrics endpoints.
```

Comment:
The doc string should have a `Raises` section listing the names of exceptions that can be raised with a brief statement of the conditions under which they are raised.